### ISSUE TITLE
Adding Stardog as target graph database to write SEPSES kg triples

### ISSUE THREAD

**Issue by Radu3000** [sepses/cyber-kg-converter#2]
What do we need to do to be able to write triples to Stardog?

I have posted a related question at https://community.stardog.com/t/sepses-kg-engine-to-write-triples-to-stardog/2251

So far I have tried:
#================SPARQL Endpoint=========================
SparqlEndpoint=https://host:port/Test2/query
#Triplestore=fuseki;virtuoso;stardog;dummy
Triplestore=stardog
UseAuth=true
...

**Comment by fekaputra** [comment]
Hi, we have an interface that needs to be implemented for each triplestore that you are using `https://github.com/sepses/cyber-kg-converter/blob/master/src/main/java/ac/at/tuwien/ifs/sepses/storage/Storage.java`. 

We have the implementation for Jena Fuseki  (`https://github.com/sepses/cyber-kg-converter/blob/master/src/main/java/ac/at/tuwien/ifs/sepses/storage/impl/FusekiStorage.java`) and Virtuoso (`https://github.com/sepses/cyber-kg-converter/blob/master/src/main/java/ac/at/tuwien/ifs/sepses/storage/impl/VirtuosoStorage.java`), but yet to create the implementation for Stardog. Since this is open-source software, it would be encouraged that you write the implementation for stardog (and make a PR afterward).

**Comment by jbalint** [comment]
@Radu3000  assuming the SPARQL `INSERT` statements will work already with the Fuseki implementation, you can just add `s-put` and `s-post` scripts which load the data into Stardog instead of Fuseki. no code changes needed

**Comment by Radu3000** [comment]
@jbalint - fuseki s-post script has over 725 lines of code ... I don't think you are suggesting to re-write it for Stardog use... ;)  ... it would be like re-writing "stardog-admin import" command ...

Maybe API calls is a better route?

My 2c

**Comment by jbalint** [comment]
@Radu3000 no, I am not suggesting you rewrite anything. I am suggesting you create a script called `s-post` where you just call `stardog data add` with the endpoint, named graph, and filename... a 1-liner
